nengo_loihi/tests/test_learning.py

```diff
@@ -47,13 +48,15 @@ def pes_network(
     return model, probes
 
 
-@pytest.mark.parametrize('n_per_dim', [120, 200])
-@pytest.mark.parametrize('dims', [1, 3])
-def test_pes_comm_channel(allclose, plt, seed, Simulator, n_per_dim, dims):
+def test_pes_comm_channel(allclose, plt, seed, Simulator):
+    n_per_dim = 120
+    dims = 3
     tau = 0.01
-    model, probes = pes_network(n_per_dim, dims, seed, learn_synapse=tau)
+    simtime = 0.5
+    model, probes = pes_network(
+        n_per_dim, dims, seed, learn_synapse=tau,
+        learning_rule_type=nengo.PES(learning_rate=1e-2), period=simtime)
```

Comment:
You know the learning code better than me, I just wasn't clear what the 1D case was testing that the 3D case wasn't (i.e., what kind of bugs might be caught by the 1D case that wouldn't be caught by the 3D case).  Similarly with the 200 neurons vs 120 neurons, I wasn't sure what different conditions were being tested by those different neuron numbers.

Reply:
I'm not sure what the 1-D case might catch, but sometimes things are different between 1-D and 3-D (e.g. if the error that we send to the chip somehow ends up as a numpy scalar instead of a vector, that could cause a bug). So I think it's worthwhile to test both.

As for the number of neurons, that's less important. I do like varying things just to catch weird things like the 300 bug (really, we should maybe pick random numbers of neurons for our tests, or something), but since this is neurons per dim anyway, we could have it the same for both 1 and 3 dims and have different numbers of neurons.

Reply:
Sounds good, added `dims=1,3` cases back in